generator/internal/cc_stub_generator.cc

```diff
@@ -45,28 +45,28 @@ bool GenerateClientStubCC(pb::ServiceDescriptor const* service,
   p->Print(vars,
            "// Generated by the GAPIC C++ plugin.\n"
            "// If you make any local changes, they will be lost.\n"
-           "// source: $proto_file_name$\n");
+           "// source: $proto_file_name$\n\n");
 
   for (auto const& include : includes) {
-    p->Print("\n#include $include$", "include", include);
+    p->Print("#include $include$\n", "include", include);
   }
   for (auto const& nspace : namespaces) {
-    p->Print("\nnamespace $namespace$ {", "namespace", nspace);
+    p->Print("namespace $namespace$ {\n", "namespace", nspace);
   }
 
   p->Print("\n");
 
   DataModel::PrintMethods(
       service, vars, p,
-      "\n\ngrpc::Status\n$stub_class_name$::$method_name$("
-      "grpc::ClientContext*,"
-      "\n    $request_object$ const&,"
-      "\n    $response_object$*) {"
-      "\n    return grpc::Status(grpc::StatusCode::kUnimplemented,"
-      "\n        \"$method_name$ not implemented\");\n}",
+      "grpc::Status\n$stub_class_name$::$method_name$(\n"
```

Comment:
Why have a line break in the middle of this line? If we do want it, then make `$stub_...` start a new line.

Reply:
The `$request` and `$response` types are used fully qualified, so they can be very long, which means that `StatusOr<$response>` should be on its own line when used in function declarations and definitions.
In this case, I wanted to maintain consistency with that pattern and shave a few characters off the signature if `$class_name` or `$method_name` were very long.

Reply:
That seems fine, but then let's break the string so that each string has just one `\n` character, at the end, so that the template looks as much like the generated code as possible.

Reply:
Done.